src/core_plugins/kibana/public/visualize/embeddable/visualize_template.html

```diff
@@ -1,11 +0,0 @@
-<visualize
-  show-spy-panel="getShouldShowSpyPane()"
-  saved-obj="savedObj"
-  app-state="appState"
-  ui-state="uiState"
-  data-shared-item
-  data-title="{{sharedItemTitle}}"
-  data-description="{{savedObj.description}}"
-  render-counter
```

Comment:
is render-counter still used ? if not, why not ?

Reply:
I assumed that it's still used. The visualize loader adds this to every visualization (see [template](https://github.com/elastic/kibana/blob/master/src/ui/public/visualize/loader/loader_template.html)). Since this directive has a rather small overhead I thought it's not bad having this on any embedded visualization, and I didn't want to implement a generic param to add angular directives to the `<visualize>` via the loader, since I considered this worth (regarding possible performance impacts and security).

Reply:
we still need render-counter for now, for reporting. however, @kobelb has an idea to remove this in favor of a render-state flag. he is looking into it and will put up a proposal.